.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'T: task, P: normal'
+assignees: ''
+
+---
+
+**Rationale**
+The reason for the enhancement.
+
+**Description**
+A clear and concise description (design) of what you want to happen.
```

Comment:
I always struggle with the Rationale and Description headings. It seems like I'm always repeating myself in these two.

Reply:
Fair - perhaps we can collapse it down in the case of enhancements. However, now that we are splitting, let me attempt to clarify what the difference is specifically here. It really is "reason" and "design". When it comes to change management, you are supposed to document why you are changing something when it's not broken. This isn't a bug report after all. So why do you want to change "this", or add something to it? Perhaps because you have a milestone? Perhaps because you want to improve something? Everything else should go in the design. I would entertain making the header reason and being more specific (perhaps using some of the wording, I just used to explain what goes there). 

In the existing template, it is really confusing because there are these "or" statements in there to handle the two different types of issues that NQA-1 likes us to document.

Reply:
I like your suggestion of replacing this with 'Reason/Design'. I guess 'Rationale' and 'Reason' really mean the same thing, so I'm OK with either one. I definitely like replacing 'Description' with 'Design' -- that makes it more clear what we are supposed to put in there.